stacker/commands/stacker/graph.py

```diff
@@ -0,0 +1,31 @@
+"""Prints the the relationships between steps as a graph.
+
+"""
+
+from .base import BaseCommand
+from ...actions import graph
+
+
+class Graph(BaseCommand):
+
+    name = "graph"
+    description = __doc__
+
+    def add_arguments(self, parser):
+        super(Graph, self).add_arguments(parser)
+        parser.add_argument("-f", "--format", default="dot",
+                            choices=list(graph.FORMATTERS.iterkeys()),
+                            help="The format to print the graph in.")
+        parser.add_argument("--reduce", action="store_true",
```

Comment:
Maybe throw a note in here that while this produces a less noisy/better graph diagram, it's slower.